pom.xml

```diff
@@ -150,6 +150,7 @@
     <shrinkwrap.version>3.0.1</shrinkwrap.version>
     <slf4j.version>1.7.25</slf4j.version>
     <spotbugs-annotations.version>3.1.1</spotbugs-annotations.version>
+    <testcontainers.version>1.7.0</testcontainers.version>
```

Comment:
@sijie 
2 FYIs:
1) there is `org.testcontainers:testcontainers-bom` with all modules (no other dependencies listed)
2) Latest version is 1.7.3 and highly recommended 👍

Reply:
@bsideup awesome! thank you for the information. I will update them.

Reply:
@sijie feel free to ping me anytime about TC, happy to help :)

FYI I was even planning to add Pulsar module to TC ;)